### ISSUE TITLE
terms that require IRIs

### ISSUE THREAD

**Issue by jdeck88** [futres/template#35]
The following terms invoke the dwc IRI namespace:
bedURI, groupURI, formationURI,localChronologyURI, memberURI 
and some more terms that suggest URI terms:
sitenameURI, localChronologyURI, formationURI

I'm not entirely convinced that we should goto the trouble of adding these terms to our futres template.  In general, i think IRI terms are a positive thing, but handling supplied IRIs properly is  problematic in terms of implementation.  E.g. I won't be able to guarantee that supplied IRIs are resolvable.  Also, internally, i'll only be using the IRI as a data property and cannot use the IRI itself to denote a class property.  In plain english any supplied IRI will just be another attribute, one that is more difficult to handle than something written out and redundant with the equivalent non-IRI term.  Also, the terms that have IRI equivalents seem a bit random-- there doesn't seem to be a systematic plan for their intake.  Finally, how much of the data we are acquiring will actually have this data?  Ok-- those are my arguments for dropping them, but willing to be convinced otherwise.

**Comment by megbalk** [comment]
if have an URI for a term, then recognize it in a parser and not as a separate column in template to separate out literals from those with URIs (e.g., bed and bedURI).
